src/views/SignIn.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+    <div class="row">
+  <div class="signIn w-100">
+    <form>
+      <b-col>
+        <b-form-group id="input-group-first-name" label="Your Firstname:" label-for="input-first-name">
+          <b-form-input
+            id="input-first-name"
+            class="w-100 input-styling"
+            type="text"
+            v-model="signIn.firstName"
+            required
+            placeholder="Max">
+          </b-form-input>
+        </b-form-group>
+      </b-col>
+      <b-col>
+      <b-form-group id="input-group-last-name" label="Your Lastname:" label-for="input-last-name">
+        <b-form-input
+          id="input-last-name"
+          type="text"
+          class="w-100 input-styling"
+          v-model="signIn.lastName"
+          required
+          placeholder="Mustermann">
+        </b-form-input>
+      </b-form-group>
+      </b-col>
+      <b-col>
+      <b-form-group id="input-group-initials" label="Your Initials:" label-for="input-initials">
+        <b-form-input
+          id="input-initials"
+          type="text"
+          class="w-100 input-styling"
+          v-model="signIn.initials"
+          required
+          placeholder="mmn"></b-form-input>
+      </b-form-group>
```

Comment:
```suggestion
        <b-form-input
          id="input-initials"
          type="text"
          class="w-100 input-styling"
          v-model="signIn.initials"
          required>
          </b-form-input>
          <b-form-text id="initials-help-block">
            Your initials will be used as a username. This in connection with your email ensures a uniqueness.
            Example: mmn
    </b-form-text>
      </b-form-group>
```

Ich würde hier auf den Platzhalter verzichten und einen Text darunter schreiben (klein in grau) wie der b-form-text eben ist. In diesem "Help-Block" würde ich erklären warum hier initials benutzt werden und wie diese z.B aussehen können.